main.pm

```diff
@@ -719,6 +719,9 @@ sub load_x11tests() {
         && !system_is_livesystem)
     {
         loadtest "x11/gnucash.pm";
+        loadtest "x11/xchat.pm";
+        loadtest "x11/hexchat.pm";
+        loadtest "x11/vlc.pm";
```

Comment:
do you check xchat and vlc are installed by default x11/gnome/kde pattern? they're on the online repo but I didn't see those are trying to install inside of test, ie. ensure_installed. so if they aren't installed in default pattern the tests won't work.